features/steps/flaw_detail.py

```diff
@@ -573,4 +581,18 @@ def step_impl(context):
     go_to_specific_flaw_detail_page(context.browser)
     flaw_detail_page = FlawDetailPage(context.browser)
     flaw_detail_page.check_value_not_exist(context.ps_component)
+    context.browser.quit()
+
+@when("I click 'delete' button of an affect")
+def step_impl(context):
+    context.ps_component = click_affect_delete_btn(context)
+    # Click the delete button of the affect
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.click_button_with_js('affectRecoverBtn')
+
+@then("I could 'recover' the affect that I tried to delete above")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.check_value_exist(context.ps_component)
```

Comment:
Is it sufficient to only check one field value.

Reply:
Updated to check the module and component.